homeassistant/components/ruckus_unleashed/device_tracker.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Ruckus Unleashed devices."""
+from typing import Optional
+
+from homeassistant.components.device_tracker import SOURCE_TYPE_ROUTER
+from homeassistant.components.device_tracker.config_entry import ScannerEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.helpers import entity_registry
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import CLIENTS, COORDINATOR, DOMAIN, UNDO_UPDATE_LISTENERS
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up device tracker for Ruckus Unleashed component."""
+    coordinator = hass.data[DOMAIN][entry.entry_id][COORDINATOR]
+
+    tracked = set()
+
+    def router_update():
```

Comment:
You need mark this also callback

Reply:
Good catch, added the `@callback` decorator